website/src/app/blog/jan-2024-product-update/readme.mdx

```diff
@@ -0,0 +1,75 @@
+_This is the first post of our new product newsletter aimed at providing regular
+updates on the Firezone product. We'll use these posts going forward to announce
+new features, major updates to existing features, and any other product-related
+news._
+
+Happy new year from the Firezone team!
+
+After a long year of building, we're incredibly excited to announce 1.0 beta
+testing for Apple and Android platforms. Firezone 1.0 is an entirely new product
+with a brand new architecture that includes many of the features you've been
+asking for. To summarize just a few:
+
+- **Native clients for every major platform** (macOS, iOS, Android, Linux,
+  Windows) that authenticate against your configured identity provider
+- **Automatic firewall hole punching** with intelligent routing that ensures
+  connections always pick the shortest path between clients and gateways
+- **Highly available gateways** with support for automatic load balancing and
+  failover
+- **Group-based, granular access policies** for managing access to IPs, CIDR
+  ranges, and even DNS-based resources like `gitlab.company.com`
+- **Automatic user/group sync** (Google Workspace currently supported)
+
+Our commitment to transparency remains the same: Firezone is still (and always
+will be) a 100% open-source product. Anyone can audit the
+[codebase](https://github.com/firezone/firezone) and verify that the product
+works the way we claim it does.
+
+## Firezone 1.0 beta testing on Apple and Android
+
+If you're on a recent Apple or Android device, you're in luck! These platforms
+are available for beta testing today. Apply for the closed beta
+[here](/product/early-access), and we'll be in touch. We're currently
+prioritizing teams and businesses at this time, but plan to roll out invites to
+individuals and homelabbers along with Windows and Linux betas over the next few
+weeks.
+
+To beta test 1.0, you'll need one of the following:
+
+- Apple silicon or Intel Mac running **macOS 12** or later
```

Comment:
Are we officially supporting macOS 12?  For some reason I was thinking we were only going to support 13 for now?  But, I may have missed something.

Reply:
12 should be working now that Split DNS isn't performed in the macOS side of things. There may still be a bug with network extension prompting for Keychain access, but the DNS should in Theory be fixed.

So far, every beta tester has been running macOS 14 so I wanted to cast a wide net and see what we get w.r.t. OS version requirements.

Reply:
cc @AndrewDryga -- have you tried with the later clients? Are you still on 12?

Reply:
@jamilbk yeah, the latest testflight have whole set of new issues:

1. System DNS resolver is not broken anymore but traffic doesn't go through the tunnel at all.

2. When I click "Disconnect and Quit" and then quickly open Firezone again I'm locked out from connecting button:
<img width="243" alt="Screenshot 2024-01-11 at 11 45 14" src="https://github.com/firezone/firezone/assets/1877644/09e302fa-ed5b-4ade-b52e-06e12dfd2ad6">

After reboot it's fine, but I need to sign in again:
<img width="170" alt="Screenshot 2024-01-11 at 11 46 25" src="https://github.com/firezone/firezone/assets/1877644/10d1d46a-0de9-419b-b7a0-06e5e11ebd67">

3. UI doesn't look like it's rendered properly for me:
<img width="512" alt="Screenshot 2024-01-11 at 11 41 54" src="https://github.com/firezone/firezone/assets/1877644/2d7451dc-6846-45ec-a821-4754f9b333a1">
<img width="338" alt="Screenshot 2024-01-11 at 11 44 00" src="https://github.com/firezone/firezone/assets/1877644/2cf43f24-c47d-4797-95f1-fe4aee994630">